README.md

```diff
@@ -31,45 +22,50 @@ Technical documentation specific to the Sinopia BIBFRAME Editor may also be foun
 2.  Install [npm](https://www.npmjs.com/)
 3.  Run `npm init`, and follow the instructions that appear.
 4.  Get latest npm: `npm install -g npm@latest`
-5.  Run `npm install grunt-cli` to install grunt-cli.
-6.  Run `npm install`. This installs everything needed for the build to run successfully.
-7.  Run `npm run build` to build the code.  (TEMP: Run `npm run grunt` or `grunt` to build the BFE code.)
+5.  Run `npm run build` to build the code.
+
+### Installing BFE code
+1.  Run `npm install grunt-cli` to install grunt-cli.
```

Comment:
do we want to just add grunt-cli to the dev dependencies?  Would make the instructions simpler.